static/js/components/ProfileImageUploader.js

```diff
@@ -76,7 +76,7 @@ const ProfileImageUploader = ({
         type='button'
         className={`save-button ${photo ? 'primary-button' : 'secondary-button disabled'}`}
         key="save"
-        onClick={updateUserPhoto}>
+        onClick={photo ? updateUserPhoto : undefined}>
```

Comment:
I think that under the hood `props` are just objects, so `undefined` looks like the button didn't receive an `onClick` handler at all.